src/elements/radioButton/radioButton.spec.js

```diff
@@ -0,0 +1,77 @@
+describe('Semantic-UI: Elements - smRadioButton', function() {
+  'use strict';
+
+  var $scope, $compile, html;
+
+  beforeEach(module('semantic.ui.elements.radioButton'));
+
+  beforeEach(inject(function($rootScope, _$compile_) {
+    $scope = $rootScope.$new();
+    $compile = _$compile_;
+  }));
+
+  function renderRadioBtns($scope) {
+    html = angular.element([
+      '<sm-radio-group ng-model="radio.value">',
+        '<sm-radio-button value="1">One</sm-radio-button>',
+        '<sm-radio-button value="2">Two</sm-radio-button>',
+        '<sm-radio-button value="3">Three</sm-radio-button>',
+      '</sm-radio-group>'].join('')
```

Comment:
@caitp, can you make a refresh in here please. What sort of helper function did you mean ? and also my initial question is still needs an answer :)